lib/health_checks/mongoid_custom_client.rb

```diff
@@ -0,0 +1,13 @@
+module MongoidCustomClient
+  module_function
```

Comment:
Use `module_function` only when you have more than one function in a module or when you really plan on using this as both a function on an instance and on the module itself

Reply:
ook, so then a regular module?

Reply:
???

Reply:
I mean, I can do it like a class with a static method, but since a module is simpler that a class and will do the same job, I will just leave it as a module and import it where it is used

Reply:
What? For homework: classes and modules in Ruby. Just do
```
module MongoidCustomClient
  def self.create
    ...
  end
end

# use it anywhere
MongoidCustomClient.create
```

Reply:
aaa, didn't know I can prepend self to module methods. For homework it will be then